auth/src/main/res/drawable/edit_text_cursor_custom.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Don't our stuff "custom" but "Settle Up" so we know it's specific to our app. edit_text_cursor_settleup